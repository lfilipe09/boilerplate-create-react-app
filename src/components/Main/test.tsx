import { screen } from '@testing-library/react'
import React from 'react'
import { renderWithTheme } from 'utils/tests/test-utils'

import Main from '.'

describe('<Main />', () => {
  it('should render the heading', () => {
    const { container } = renderWithTheme(<Main />)

    expect(
      screen.getByRole('heading', { name: /react avan√ßado/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
