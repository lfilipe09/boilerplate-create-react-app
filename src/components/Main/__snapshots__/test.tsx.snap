// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
<main
  class="sc-bczRLJ byjVY"
>
  <img
    alt="Imagem de um átomo e React Avançado escrito ao lado."
    class="sc-gsnTZi ebxzkr"
    src="/logo192.png"
  />
  <h1
    class="sc-dkzDqf BthzW"
  >
    React Avançado
  </h1>
  <h2
    class="sc-hKMtZM cLAtTB"
  >
    TypeScript, ReactJS, NextJS e Styled Components
  </h2>
  <img
    alt="Um desenvolvedor de frente para uma tela com código."
    class="sc-eCYdqJ cQnCtQ"
    src="/img/hero-illustration.svg"
  />
</main>
`
